src/cart/crt_init.c

```diff
@@ -118,6 +120,20 @@ prov_data_init(struct crt_prov_gdata *prov_data, crt_provider_t provider,
 	if (rc != 0)
 		return rc;
 
+	/* Set max number of contexts. Defaults to the number of cores */
+	ctx_num = 0;
+	d_getenv_int("CRT_CTX_NUM", &ctx_num);
```

Comment:
do you need to update README.env ?

Reply:
good point. i ll push doc-only change separately for this